build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
```

Comment:
> But what's the process ?

The process is independent of the output. Think of this file in the context of a infrastructure where we have completely separate build + publish steps. This file just defines what publish events occur when we decide it's time to publish. 

> Is your thinking that once we've transitioned to shipping mode we're always doing that, even for intermediate builds?

Yes. I think this is our long term location.